gammapy/estimators/points/sensitivity.py

```diff
@@ -49,6 +53,10 @@ def __init__(
             spectral_model = PowerLawSpectralModel(
                 index=2, amplitude="1 cm-2 s-1 TeV-1"
             )
+        if spectral_model.integral(1 * u.TeV, 10 * u.TeV).value < 0.0:
```

Comment:
I would rather move that in the run method and evaluate at the energies relevant for the dataset